apps/remote_control/lib/lexical/remote_control/dispatch/handlers/indexing.ex

```diff
@@ -24,9 +25,10 @@ defmodule Lexical.RemoteControl.Dispatch.Handlers.Indexing do
     {:ok, state}
   end
 
-  defp reindex(%Document{} = document, quoted_ast) do
-    with :ok <- ensure_latest_version(document),
-         {:ok, entries} <- Indexer.Quoted.index(document, quoted_ast) do
+  defp reindex(uri) do
+    with {:ok, {%Document{} = document, %Analysis{} = analysis}} <-
```

Comment:
I think we should drop the nested tuples here and return `{:ok, %Document{}, %Analysis{}}` Thoughts?